src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -6,7 +6,8 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\packages\Avalonia\Avalonia.csproj" />
-    <PackageReference Include="Avalonia.Angle.Windows.Natives" Version="2.1.0.2019013001" />
+    <PackageReference Include="Avalonia.Angle.Windows.Natives" Version="2.1.0.2020091801" />
+    <PackageReference Include="Microsoft.Windows.SDK.NET" Version="10.0.18362.3-preview" />
```

Comment:
The Microsoft.Windows.SDK.NET adds 60MB Microsoft.Windows.SDK.NET.dll into publish folder.

Reply:
Seems to be marked as `PrivateAssets="All"`
> The Windows SDK available as a NuGet package for more seamless acquisition and CI/CD integration. This package is designed for .NET 5 applications.
https://www.nuget.org/packages/microsoft.windows.sdk.net

Reply:
I wonder why, isn't it just C# binding to SDK?

Reply:
The original dll is 25MB, I think PublishReadyToRun=True made it even bigger.

Reply:
@jkoritzinsky can tell us!

Reply:
It takes a lot of code to successfully implement WinRT, and the Windows SDK is massive. The team is working on slimming it down (I have a PR open right now to cut the size), but it's still going to be pretty large until it isn't shipped as a monolith.

Reply:
We dont get Acrylic material without this im afraid!

Reply:
@danwalmsley can we make acrylic with WinUI.Comp package optional then?
So user will need to add Avalonia.WinUI package and include it in builder with .UseWinUIComposition() ?

Reply:
We could... but is the extra file size such a huge problem? it can be stripped away!

Reply:
It would mean Acrylic on windows is not available at all until this package is added, which I think would be confusing... and also as @jkoritzinsky this issue is going to be solved in future... this is just a temporary solution until the frameworks catches up.

Reply:
ok so @kekekeks and I have agreed to move to using a native shim, that will be a few kb to provide the winrt interop.

This will be done with a separate PR.